code/_onclick/other_mobs.dm

```diff
@@ -50,6 +50,8 @@
 	. = FALSE
 	if(!(interaction_flags_atom & INTERACT_ATOM_NO_FINGERPRINT_ATTACK_HAND))
 		add_fingerprint(user)
+	if(atom_storage && atom_storage.handle_attack(user))
+		. = TRUE
 	if(SEND_SIGNAL(src, COMSIG_ATOM_ATTACK_HAND, user, modifiers) & COMPONENT_CANCEL_ATTACK_CHAIN)
 		. = TRUE
```

Comment:
why not just use the signal for this sort of thing

Reply:
part of the whole reason i'm datumizing storage is to remove the use of signals

Reply:
see above